test/e2e/tests/vault-corruption/vault-corruption.spec.ts

```diff
@@ -122,6 +122,7 @@ describe('Vault Corruption', function () {
   async function waitForVaultRestorePage(driver: Driver) {
     await driver.waitUntil(
       async () => {
+        await driver.openNewPage('about:blank');
```

Comment:
we make sure we open the page every time we try the condition, otherwise, the page is automatically close and then we get the webview not found